File: javascript01-bit/student-selector.js
"use strict"

const students = [
  "김미연", "우인재", "박규호", "김정준", "노완진", "한홍근", "임의수",
  "송수정", "홍효선", "고진서", "홍순민", "이수은",
  "김대연", "조성규", "이유운", "이윤민", "김승원", "양승열",
  "권봉수", "신용주", "김승민", "정성룡", "박은비", "신선아", "서흥원", "김민섭"
];

var count = 10;
var timer = setInterval(function() {
  var no = parseInt(Math.random() * 26);
  if (count == 0 ) {
    console.log(students[no]);
    clearInterval(timer);
    return;
  }
  console.log(count--);
}, 1000);

File: java01-bit/src/StudentSelector.java
import java.util.ArrayList;
import java.util.Scanner;

public class StudentSelector {

  public static void main(String[] args) throws Exception {
    Scanner keyScan = new Scanner(System.in);
    
    //String[] names = {"양승렬","박규호","권봉수","신용주","송수정","신선아","김정준","우인재"};
    //String[] names = {"김정준","권봉수","김승원","홍순민","고진서","김미연",
    //    "김대연","정성룡","김민섭","서흥원","김승민"};
    String[] names = {"권봉수","김승원","고진서","정성룡","서흥원","홍길동"};
        
    ArrayList<String> students = new ArrayList<>();
    for (String name : names) {
      students.add(name);
    }
    
    for (int i = 0; i < 6; i++) {
      keyScan.nextLine();
      int count = (int)(Math.random() * 1000);
      for (int x = 0; x < count; x++) { 
        if (x % 10 == 0) System.out.print("."); 
      }
      int no = (int)(Math.random() * students.size());
      System.out.println(students.remove(no));
      
    }

  }

}
